packages/geocoder/src/index.story.tsx

```diff
@@ -39,21 +48,23 @@ const GeocoderTester = ({
     reverseUseFeatureCollection
   });
 
+  const searchObj: AnyGeocoderQuery = {
+    text: searchTerm
+  };
+
+  if (enableBoundary) {
+    searchObj.boundary = { rect: boundary };
+  }
+  if (enableFocusPoint) {
+    searchObj.point = focusPoint;
+    searchObj.focusPoint = focusPoint;
```

Comment:
We update `point` and `focusPoint`?

Reply:
I think it's because point is used for reverse and focusPoint is used for others. But since the story just has the one field I just set both of them in the query.

Reply:
That makes sense thanks